packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/mtom/parser.ts

```diff
@@ -77,14 +77,22 @@ export async function parseMtomResponse(
 }
 
 //eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function getBoundaryFromMtomResponse(contentType: any): string {
+export function getBoundaryFromMtomResponse(contentType: any): string | undefined {
   const parsedContentType = MIMEType.parse(contentType);
   if (!parsedContentType) {
     throw new Error("Parsing of content type failed");
   }
   const boundary = parsedContentType.parameters.get("boundary");
-  if (!boundary) {
-    throw new Error("No boundary parameter found in content type.");
-  }
   return boundary;
 }
+
+export function convertSoapResponseToMtomResponse(buffer: Buffer): IMTOMAttachments {
```

Comment:
so testing on staging, where implementers dont follow IHE spec, works. 
they return non mtom responses to mtom requests which is wrong but we should still handle it